requirements.txt

```diff
@@ -1,6 +1,6 @@
 setuptools
 flask>=1.1.2
-wikidataintegrator==0.6.5
+wikidataintegrator==0.8.4
```

Comment:
Latest WDI needed because the old version doesn't support all the statement data types (was missing Lexeme and was throwing an error trying to render items that have statements using new types)

Reply:
My hope here is that nothing's changed with OAuth. I doubt it as I suspect that they have an "if it ain't broke" approach.

Reply:
Good call! I've been testing it locally and our existing code for Oauth has been working so far after the update. I'll let you know if I see anything off